### PR TITLE
C357987 Verify Users Patron group bulk edit -- in app approach

### PR DESCRIPTION
**Pull request by MaksimTerekhov**

### CODE REVIEW

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-update-users.cy.js [hunk 1]
```diff
@@ -7,6 +7,8 @@ import getRandomPostfix from '../../../support/utils/stringTools';
 import devTeams from '../../../support/dictionary/devTeams';
 import BulkEditActions from '../../../support/fragments/bulk-edit/bulk-edit-actions';
 import Users from '../../../support/fragments/users/users';
+import UsersSearchPane from "../../../support/fragments/users/usersSearchPane";
```
Comment (IhorBohdan) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-update-users.cy.js:10]: Fix linter issues

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-update-users.cy.js [hunk 2]
```diff
@@ -61,5 +63,24 @@ describe('bulk-edit', () => {
       BulkEditSearchPane.selectRecordIdentifier('Usernames');
       BulkEditSearchPane.verifyDragNDropUsernamesArea();
     });
+
+    it('C357987 Verify Users Patron group bulk edit -- in app approach (firebird)', { tags: [testTypes.smoke, devTeams.firebird] }, () => {
+      BulkEditSearchPane.selectRecordIdentifier('User UUIDs');
+
+      BulkEditSearchPane.uploadFile(userUUIDsFileName);
+      BulkEditSearchPane.waitFileUploading()
```
Comment (IhorBohdan) [inline cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-update-users.cy.js:71]: You missd ";"

**Review by IhorBohdan — APPROVED** [review]

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-update-users.cy.js (hunk 1 above)
Comment (MaksimTerekhov) [reply cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-update-users.cy.js]: thx, fixed

File: cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-update-users.cy.js (hunk 2 above)
Comment (MaksimTerekhov) [reply cypress/e2e/bulk-edit/in-app/bulk-edit-in-app-update-users.cy.js]: thx, fixed

**Review by TetianaParanich — APPROVED** [review]

**Pull request merged**